src/Snippets/onecc.json

```diff
@@ -0,0 +1,81 @@
+{
+  "onecc": {
+    "prefix": "onecc",
+    "body": [
+      "[onecc]",
+      "one-import-tf=${1:True}",
+      "one-import-tflite=${2:False}",
+      "one-import-bcq=${3:False}",
+      "one-import-onnx=${4:False}",
+      "one-optimize=${5:False}",
+      "one-quantize=${6:False}",
+      "one-pack=${7:False}",
+      "one-codegen=${8:False}",
+      "one-profile=${9:False}",
+      "",
+      "[one-import-tf]",
+      "input_path=$10",
+      "output_path=$11",
+      "input_arrays=$12",
+      "input_shapes=$13",
+      "output_arrays=$14",
+      "converter_version=$15",
+      "model_format=$16",
+      "save_intermediate=$17",
+      "",
+      "[one-import-tflite]",
+      "input_path=$18",
+      "output_path=$19",
+      "",
+      "[one-import-bcq]",
+      "input_path=$20",
+      "output_path=$21",
+      "input_arrays=$22",
+      "input_shapes=$23",
+      "output_arrays=$24",
+      "converter_version=$25",
+      "",
+      "[one-import-onnx]",
+      "input_path=$26",
+      "output_path=$27",
+      "input_arrays=$28",
+      "output_arrays=$29",
+      "converter_version=$30",
+      "model_format=$31",
+      "save_intermediate=$32",
+      "",
+      "[one-optimize]",
+      "input_path=$33",
+      "output_path=$34",
+      "",
+      "[one-quantize]",
+      "input_path=$35",
+      "output_path=$36",
+      "input_data=$37",
+      "input_data_format=$38",
+      "generate_profile_data=$39",
+      "input_dtype=$40",
+      "quantized_dtype=$41",
+      "granularity=$42",
+      "min_percentile=$43",
+      "max_percentile=$44",
+      "mode=$45",
+      "",
+      "[one-pack]",
+      "input_path=$46",
+      "output_path=$47",
+      "",
+      "[one-codegen]",
+      "backend=$48",
+      "command=$49",
+      "",
+      "[one-profile]",
+      "backend=$50",
+      "command=$51",
+      "$0"
+    ],
+    "description": [
+      "Run ONE driver via several commands or configuration file"
```

Comment:
This seems different from `one-build`. Could you please make two similar?

Reply:
I brought a description of each tool through the following command.

![image](https://user-images.githubusercontent.com/44386047/133574391-1e6ce800-cd93-45dd-b0ea-b44e54b5e971.png)


![image](https://user-images.githubusercontent.com/44386047/133574227-8e0ed081-5355-480a-ae92-5996a4db8de1.png)

Can I use the same one as `one-build`?
Or can you recommend something?

Reply:
As onecc is newer one, let's go with this and revise for one-build in another PR, please post :)

Reply:
or if we follow that from `one-build`, we may go as-is :)

Reply:
I didn't understand exactly.
Do I have to revise `one-build` and upload the pull request again?
Otherwise, do I have to keep both things as they are?
If we modify `one-build`, how should we modify it?

Reply:
No you don't have to modify `one-build`. You've chosen the message as-is from `one-build` so it is OK to go with it in this repo.

Reply:
I definitely understood. Thank you. :)